lib/hammer_cli_foreman_tasks/helper.rb

```diff
@@ -4,9 +4,13 @@ module Helper
     # render the progress of the task using polling to the task API
     def task_progress(task_or_id)
       task_id = task_or_id.is_a?(Hash) ? task_or_id['id'] : task_or_id
-      task_progress = TaskProgress.new(task_id) { |id| load_task(id) }
-      task_progress.render
-      task_progress.success?
+      if !task_id.empty?
+        task_progress = TaskProgress.new(task_id) { |id| load_task(id) }
+        task_progress.render
+        task_progress.success?
+      else
+	signal_usage_error(_("Please mention appropriate attribute value"))
```

Comment:
2-space indentation please

Reply:
@tstrachota have made the indentation proper.